### ISSUE TITLE
Sceneform updates and SFB binary format

### ISSUE THREAD

**Issue by StevenOttoG** [google-ar/sceneform-android-sdk#456]
Could you clarify if and how the tooling and the SFB file format changes between the different releases? Is it safe to load models created with an older version of Sceneform or should we rebuild our models with every release?

**Comment by romainguy** [comment]
@AdrianAtGoogle @dsternfeld7

**Comment by romainguy** [comment]
At the moment rebuilding models is necessary when the .sfb contains a material and the internal shader structure has changed in the rendering engine. We were recently talking about dealing with this in a better way. For instance we have a [pending renderer change](https://github.com/google/filament/pull/601) that would require materials/sfb files to be regenerated.

**Comment by StevenOttoG** [comment]
Could you maybe expose something like a SFB file format version number in the API and include changes in the release notes? This way we don't have to keep a model per Sceneform version.

**Comment by romainguy** [comment]
Yeah I agree we should at the very least let you know when a rebuild is necessary.

**Comment by StevenOttoG** [comment]
That would be great! Thanks!
